### DISCUSSION TITLE
It can be interesting to have a x-on:keydown.stop.except.esc for example

### DISCUSSION THREAD

**Discussion by joffreypersia** [alpinejs/alpine#4271] [3. Feature ideas]
Hi, 

For example in a modal, I have an input text field, I would like to block all the propagation of the letters (because of other shortcuts I might have set), but only make the propagation of Esc to close the modal. 

Today I need to dispatch an event for that, that is not a keydown event. 

PS: I don't use the keyup.stop because the shortcut listen to keydown (quicker for multi keys shortcut).

Best,

**Comment by ekwoka** [comment]
It's simple enough to just do

```
@keydown="$event.key !== 'escape' && $event.stopPropagation()"
```
